### DISCUSSION TITLE
Let's create Learn.md file

### DISCUSSION THREAD

**Discussion by Yeasir0032** [Yeasir0032/Discord-Clone#9] [Ideas]
# Learn.md file Creation
Issue : [Learn.md file issue](https://github.com/Yeasir0032/Discord-Clone/issues/4)

To create this follow the steps: 
- Create a new Learn.md file in the root directory.
- Copy the [local development](https://github.com/Yeasir0032/Discord-Clone#local-development) part of Readme file
- Add some more details.

**Comment by meAyushSharma** [comment]
I would like to work on it.

**Comment by Yeasir0032** [comment]
@meAyushSharma 
I have assigned the issue to you.

**Comment by Yeasir0032** [comment]
@meAyushSharma can you do supabase database setup in the documentation. Like the schema, tables , columns and all that?
You have to do:
- Go to **supabse dashboard**
- Create a new project
- Create tables (I will provide the sql code)
- **Learn.md** file should contain how this setup.

**Reply by meAyushSharma** [reply]
@Yeasir0032 I have done first and second points, please provide me with sql code.

**Reply by Yeasir0032** [reply]
If you can create the sql code then do it otherwise **leave it**
Here is the schema for the supabase project
![Screenshot from 2024-05-11 20-28-26](https://github.com/Yeasir0032/Discord-Clone/assets/124646897/0b25d183-ad93-4ac0-ba0d-e106312d402d)
#### If you don't want to do it just leave and make the PR
